### ISSUE TITLE
Need clarity on OSM governance and CLA

### ISSUE THREAD

**Issue by mudit-01** [openservicemesh/osm#1999]
@phillipgibson  here are few points that OSM documents should cover:-
- Page on governance hierarchy like other CNCF sandbox projects
- More clarification on the corporate CLA/DCO like other sandbox projects as current one is not so clear

**Comment by phillipgibson** [comment]
Thanks @mudit-01 I moved this to our current milestone to be completed.

**Comment by bridgetkromhout** [comment]
From CNCF Slack:
```
Chris Aniszczyk  1:20 PM
It's up to the OSM maintainers but they need to choose between the CLA / DCO
```

**Comment by bridgetkromhout** [comment]
This is what we need to update: https://github.com/openservicemesh/osm/blob/main/CONTRIBUTING.md#contributor-license-agreement
https://github.com/openservicemesh/osm-www/blob/main/CONTRIBUTING.md#contributor-license-agreement

**Comment by bridgetkromhout** [comment]
TLDR: I think you need to open a ticket with the CNCF to get the URL to set up the CLA under CNCF.

**Comment by phillipgibson** [comment]
DCO guidance has been added to the contributing.md doc, DCO bot has been added to the org level, and original CLA requirements have been removed from branch rules.

**Comment by mudit-01** [comment]
@phillipgibson the governance part is still missing like other projects have a GOVERNANCE.md. Any updates on it?

**Comment by phillipgibson** [comment]
@mudit-01 the OSM repo does not contain a governance.md file. OSM is a CNCF sandbox project and falls under the CNCF governance for projects. Can you provide more specifics of what you are looking for? Thanks.

**Comment by mudit-01** [comment]
@phillipgibson I was searching for the requisites for becoming maintainers usually they are are documented governance.md

**Comment by phillipgibson** [comment]
Understood. I will re-open this issue and get a draft of maintainers/contributors guidance in a governance.md file. Thanks for clarifying.

**Comment by phillipgibson** [comment]
At the current state of a CNCF sandbox project we do not need the governance doc completed at this point. We will be working on drafting the governance.md doc in the near future.
